src/gg/codie/mineonline/server/MinecraftServerLauncher.java

```diff
@@ -55,8 +57,20 @@ public MinecraftServerLauncher(String[] args) throws Exception {
             }
         }
 
-        if (serverProperties.discordToken() != null && serverProperties.discordChan() != null) { // Create the discord bot if token and channel are present
-            discord = new DiscordChatBridge(minecraftVersion.name, serverProperties.discordChan(), serverProperties.discordToken(), serverProperties.discordWebhookUrl(), new MessageRecievedListener() {
+        if (minecraftVersion != null)
+            ver = minecraftVersion.name;
+        else
+            ver = "";
```

Comment:
I think this would be better as null rather than an empty string. You'd have to check it in the DCB constructor.
As in
String versionName = Minecraft version != null ? minecraftVersion.name : null;